cpp/ql/src/Security/CWE/CWE-416/UseOfStringAfterLifetimeEnds.qhelp

```diff
@@ -8,6 +8,12 @@
 When the <code>std::string</code> object is destroyed, the pointer returned by <code>c_str</code> is no
 longer valid. If the pointer is used after the <code>std::string</code> object is destroyed, then the behavior is undefined.
 </p>
+
+<p>Typically this problem occurs when a <code>std::string</code> is returned by a function call (or overloaded operator)
+by value, and the result is not immediately stored in a variable by value or reference in a way that extends the lifetime of
+the temporary object. The resulting temporary <code>std::string</code> object is destroyed at the end of the expression
```

Comment:
What about this suggestion to make this sentence shorter? 🤔 
Feel free to ignore if you don't agree, or if it doesn't work here 😅 
```suggestion
the temporary object. The resulting temporary <code>std::string</code> object is destroyed at the end of the containing expression
```